StPicoEvent/StPicoTrack.h

```diff
@@ -45,6 +45,11 @@ class StPicoTrack : public TObject
   UInt_t  topologyMap(unsigned int) const;
 
   StPhysicalHelixD helix(float B) const;
+  bool isPxl1() const;
+  bool isPxl2() const;
+  bool isIst() const;
+  bool isSst() const;
+  bool isHft() const;
```

Comment:
It makes sense to name functions returning boolean values starting with either "is.." (as you did) or "has..". "isHft()" sounds ok to me, but how about "hasXxxHit()" for the other methods? It would be somewhat closer to human language.